.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve the PureScript compiler
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
```

Comment:
Maybe just "Description" if this is the header we want in the issue itself?

Reply:
Sounds good.
```suggestion
## Description
```